### ISSUE TITLE
Step indicator is not in the reading order [iOS]

### ISSUE THREAD

**Issue by willslone** [covid-alert-ny/covid-green-app#195]
The step indicator found on the “How it works” screens is not included in the reading order. The visual information it provides is not available to VoiceOver users. I would suggest including it in the reading order and providing a proper text alternative (e.g., “Step 2 of 4”).

![Step indicator that indicates visually that the user is on step 2 of 4](https://user-images.githubusercontent.com/4437084/91868295-94ffa500-ec42-11ea-9da7-cf9a62616fa6.png)

I would categorize this issue as low severity.

**Comment by floridemai** [comment]
@willslone I've address this but using the page selector to move through pages is not possible just yet. I'd close this as it fixes the reported issue and raise the missing part in another thicket, when we do a page by page review (hopefully)
